docs/app/documentation/component-docs/localization-editor/examples/localization-editor-forms-example.component.html

```diff
@@ -0,0 +1,15 @@
+<form [formGroup]="customForm">
+    <fd-localization-editor>
+        <fd-localization-editor-main [label]="'EN'" [compact]="true">
+            <input fd-localization-editor-input formControlName="en" type="text" placeholder="EN">
+        </fd-localization-editor-main>
+        <li fd-localization-editor-element *ngFor="let key of getCustomFormKeys()">
```

Comment:
Generally should avoid referencing methods directly in the template because of how inefficient they are with the default change detection strategy.

Reply:
If you use an actual get method as in

`get customFormKeys(): someArray {}` then it would work fine